lib/rules/require-v-for-key.js

```diff
@@ -17,6 +17,7 @@ const utils = require('../utils')
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
All rules in `essential` category report rather problems.

Reply:
ohh i missed that 👍